packages/headless-wallet-button-integration/src/headless-wallet-button-integration-request-service.ts

```diff
@@ -0,0 +1,117 @@
+import { RequestSender, Response } from '@bigcommerce/request-sender';
+
+import {
+    BillingAddress,
```

Comment:
I believe these types come from REST API interface, which may differ from GraphQL, therefore we can't reuse the types. I suggest defining separate ones for GraphQL. Instead of manually defining them, consider using tools such as `graphql-codegen` that can automatically generate the types based on a GraphQL schema.